config/locales/titles.en.yml

```diff
@@ -0,0 +1,16 @@
+en:
+  titles:
+    application: Rails Base example site
+    dashboard:
+      index: Dashboard
+    devise:
```

Comment:
and here we're not using any quotes at all. :+1: on this approach: no quotes first, add quotes only when it's needed (translation contains interpolation args).